signal.go

```diff
@@ -11,17 +11,16 @@ import (
 
 // Terminate sends a SIGTERM signal to the given process under the given
 // context. This does not guarantee that the process will actually die. This
-// function does not Wait() on the given process upon sending the signal. On
-// Windows, this function sends a SIGKILL instead of SIGTERM.
+// function does not Wait() on the given process upon sending the signal.
 func Terminate(ctx context.Context, p Process) error {
-	return errors.WithStack(p.Signal(ctx, makeCompatible(syscall.SIGTERM)))
+	return errors.WithStack(p.Signal(ctx, syscall.SIGTERM))
```

Comment:
Why did you remove `makeCompatible`?

Reply:
makeCompatible is supposed to convert signals that don't work on certain platforms to ones that will (e.g. Windows doesn't accept SIGTERM so it gets converted to SIGKILL). However, the presignal triggers that run before actually sending the signal need to know the unmodified signal being requested. This just makes it clearer so that if somebody added a presignal trigger and expected that triggers to catch a particular signal, it would do so correctly without the signal being changed before the trigger runs. Once all the triggers run, the signal still gets modified before it calls the actual go library Signal() function.